src/components/StoriesGroup/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './StoriesGroup';
+export * from './types';
```

Comment:
you should alse add reexport to main index.ts

Reply:
[009fe86](https://github.com/gravity-ui/uikit/pull/387/commits/009fe86e3dcc26336bda65bc6b983645d8f1d9c1)